jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/RouteEnricher.java

```diff
@@ -121,14 +121,24 @@ private void addRoute(KubernetesListBuilder listBuilder, ServiceBuilder serviceB
         }
     }
 
-    private static RoutePort createRoutePort(ServiceBuilder serviceBuilder) {
-        RoutePort routePort = null;
-        final Integer serviceTargetPort = getTargetPortToExpose(serviceBuilder);
-        final Integer servicePort = serviceTargetPort != null ? serviceTargetPort : getPortToExpose(serviceBuilder);
-        if (servicePort != null) {
+    private static RoutePort createRoutePort(ServicePort servicePortToExpose) {
+        RoutePort routePort = new RoutePort();
+        final String sevicePortName = servicePortToExpose.getName();
```

Comment:
nit, typo in variable name
```suggestion
        final String servicePortName = servicePortToExpose.getName();
```